### PR TITLE
MMIs and positronic brains now talk like pAIs in plushies

### PR DESCRIPTION
**Pull request by beck-thompson**
## About the PR

Title.

## Why / Balance

Lot of people in the discord really wanted the feature! It makes sense as well, pAIs are basically the same as MMIs and brains.

## Technical details

Just added the necessary component so their name will get overwritten.

## Media

image

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**

:cl: Beck Thompson
- add: When MMIs and positronic brains are inserted into plushies, their names will be updated to the plushies name (Exactly the same way pAIs do).

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml [hunk 1]
```diff
@@ -49,6 +49,10 @@
     guides:
     - Cyborgs
     - Robotics
+  - type: ChangeVoiceInContainer
+    whitelist:
+      components:
+        - SecretStash
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml:55]:
```suggestion
      components:
      - SecretStash
```

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml [hunk 2]
```diff
@@ -127,3 +131,7 @@
       guides:
       - Cyborgs
       - Robotics
+    - type: ChangeVoiceInContainer
+      whitelist:
+        components:
+          - SecretStash
```
Comment (slarticodefast) [inline Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml:137]:
```suggestion
        components:
        - SecretStash
```

**Review by slarticodefast — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml (hunk 2 above)
Comment (beck-thompson) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml:137]: ok I really need to change this from the pAI because that's where I'm copying it from 😭

Could I do that in this PR as well?

Comment (slarticodefast) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml]: > Could I do that in this PR as well?

I try to sneak stuff like that into my own PRs as well and others told me not to, so probably not 😞

**Review by slarticodefast — APPROVED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml (hunk 2 above)
Comment (beck-thompson) [reply Resources/Prototypes/Entities/Objects/Specific/Robotics/mmi.yml]: > I try to sneak stuff like that into my own PRs as well and others told me not to, so probably not 😞

Makes sense I guess. I'll remember next time I touch the pai stuff 😆

**Comment by slarticodefast** [comment]
Good idea. Could you rephrase the changelog a little so it is more clear for players who don't know about the voice name change?

**Comment by chromiumboy** [comment]
I mean, sure, why not?

Tested in-game fine
